package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "check-pr-action",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "description": "This action checks the PR body and title against regular expression patterns.",
   "main": "index.js",
-  "scripts": {
```

Comment:
What do?

Reply:
This part of the `package.json` lets you run, well, scripts with `npm` (or `yarn`). E.g. `npm test`. But we don't have any and probably never will, so I'm deleting the boilerplate that was inserted when someone created this package.